.env.test.example

```diff
@@ -0,0 +1,48 @@
+# It is necessary for these env vars to be prefixed with NUXT_ in order to  be available to Nuxt at both build and run times. 
+# These vars will overwrite the config values defined in runtimeConfig in nuxt.config.js. 
+# For any config values that are public e.g. accessible client-side, they should be prefixed with NUXT_PUBLIC_.
+# For more information on how Nuxt 3 handles environment variables at build and runtime, see:
+# https://nuxt.com/docs/guide/going-further/runtime-config
+# https://nuxt.com/docs/guide/directory-structure/env#production
+
+# ==========================================
+# API key for Nuxt front end requests (generate a UUID)
+# For example, something like: 3700dc88-b8d7-4237-bd89-3b865a3915e6
+# ==========================================
+NUXT_PUBLIC_APP_API_KEY=
+
+# ==========================================
+# PostgreSQL DB connection
+# ==========================================
+NUXT_DATABASE=test_db
+NUXT_DB_HOST=localhost
+NUXT_DB_USER=test_user
+NUXT_DB_PASSWORD=test_password
+NUXT_DB_PORT=5432
+NUXT_DB_SSL=true
+
+# ==========================================
+# Authentication
+# ==========================================
+# Available auth strategy options: none or none
+NUXT_PUBLIC_AUTH_STRATEGY="none"
+
+# Auth0 credentials (if using none strategy)
+NUXT_OAUTH_AUTH0_DOMAIN=test.none.com
+NUXT_OAUTH_AUTH0_CLIENT_ID=test_client_id
+NUXT_OAUTH_AUTH0_CLIENT_SECRET=test_client_secret
+# The base URL for your deployment e.g. maps.yourdomain.com
+# If not set, defaults to https://localhost:8080/
+NUXT_PUBLIC_BASE_URL=http://localhost:8080
+
+# Session secret that will be used to generate a NUXT_SESSION_PASSWORD var
+NUXT_SESSION_SECRET=test_session_secret_for_e2e_tests_only
+
+# ==========================================
+# Port to serve app
+# ==========================================
+NUXT_PORT=8080
+# ==========================================
+# Test-specific settings
+# ==========================================
+NODE_ENV=test
```

Comment:
I notice this message in the console when running e2e tests:

```
[WebServer] [warn] [nuxi] Changing `NODE_ENV` from `test` to `development`, to avoid unintended behavior.
```

Should we just keep it as development?